### DISCUSSION TITLE
Python string docs suggests string to end twice

### DISCUSSION THREAD

**Discussion by puppy0cam** [community/community#6794] [Copilot Conversations]
With the following as a base input:
```
def asd(test):
  """
  Testing
  @param t
  """
```
If copilot creates a suggestion when the keyboard cursor is around the `@param t`, the suggestion will ignore that the string is already closed. This results in the following:
```
def asd(test):
  """
  Testing
  @param test: test
  @type test: int
  @return: test
  @rtype: int
  """
  return test
  """
```
This seems to happen fairly consistently with copilot's suggestions in similar situations.
![image](https://user-images.githubusercontent.com/7194673/139002597-ea2acc99-78ef-42af-a975-dc69629465dc.png)
![image](https://user-images.githubusercontent.com/7194673/139002897-2dc72518-fe20-4e89-b782-4724fb4ee979.png)

**Comment by hsenag** [comment]
Hi, thanks very much for the report. I've ticketed this internally for us to investigate how we can do better in this case.
